src/app/feature/consultants/pages/consultants-view/consultants-view.component.html

```diff
@@ -21,12 +21,12 @@
               </mat-panel-title>
             </mat-expansion-panel-header>
             <mat-radio-group>
-              <mat-radio-button value='All' (click)="filterChange('','','','','','','')" checked>All Practices</mat-radio-button>
-              <mat-radio-button value='Applications & Cloud Technology' (click)="filterChange('','Applications & Cloud Technology','','','','','')">Applications & Cloud Tech...</mat-radio-button>
-              <mat-radio-button value='Digital Assurance & Testing' (click)="filterChange('','Digital Assurance & Testing','','','','','')">Digital Assurance & Testing</mat-radio-button>
-              <mat-radio-button value='Digital Manufacturing' (click)="filterChange('','Digital Manufacturing','','','','','')">Digital Manufacturing</mat-radio-button>
-              <mat-radio-button value='Digital Transformation' (click)="filterChange('','Digital Transformation','','','','','')">Digital Transformation</mat-radio-button>
-              <mat-radio-button value='Insights & Data' (click)="filterChange('','Insights & Data','','','','','')">Insights & Data</mat-radio-button>
+              <mat-radio-button value='All' (click)="filterChange('','','','','','','')" checked>All Practices</mat-radio-button><br />
```

Comment:
Rather than adding all these line breaks one could add this CSS class:

`mat-radio-button {
    display: block;
}`

Up to you.